docs/source/implementations/backtesting.rst

```diff
@@ -174,4 +174,4 @@ Research Notebooks
 The following research notebooks can be used to better understand how the algorithms within this module work and how
 they can be used on real data.
 
-* `Backtesting Notebook <https://github.com/hudson-and-thames/research/blob/master/Backtesting/Backtesting.ipynb>`_
+* `Backtesting Notebook <https://github.com/hudson-and-thames/research/blob/fix_directory_structure/Backtest%20Overfitting/Backtesting.ipynb>`_
```

Comment:
This one links to `fix_directory_structure` and not `master` . Should this be fixed?

This link also has a structure `* 'Name <Link>` and other links have `.. _Name: Link`

Reply:
Ah, thanks for noticing this. I changed all other links to master but this got left out